harness/determined/cli/workspace.py

```diff
@@ -147,6 +147,98 @@ def list_pools(args: argparse.Namespace) -> None:
     )
 
 
+def list_workspace_namespace_bindings(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_ListWorkspaceNamespaceBindings(sess, id=w.id)
+    if resp.namespaceBindings:
+        print(f"workspace-namespace bindings for workspace {args.workspace_name}")
+        cluster_namespaces = []
+        namespace_bindings = resp.namespaceBindings
+        for cluster in namespace_bindings:
+            cluster_namespaces.append([cluster, namespace_bindings[cluster].namespace])
+        render.tabulate_or_csv(
+            headers=["Cluster", "Namespace"],
+            values=cluster_namespaces,
+            as_csv=False,
+        )
+    else:
+        print(f"No workspace-namespace bindings for workspace {args.workspace_name}")
+    return None
```

Comment:
For my understanding, is adding `return None` for meeting lint requirement?

Reply:
not sure, but we probably added that out of habit. Is it best practice to not return None?

Reply:
The behaviors of adding `return None` and removing it are the same. If we plan to use None later on, then adding `return None` is preferred. For example:
```
bindings = list_workspace_namespace_bindings(args)

if bindings :
    # use bindings for something
```

If the caller of `list_workspace_namespace_bindings` doesn't mean to use None, then  without adding `return None` is preferred. 

This is not a blocker and the above are not absolute rules.